app/grandchallenge/algorithms/serializers.py

```diff
@@ -107,6 +126,7 @@ class JobSerializer(serializers.ModelSerializer):
 
     algorithm_image = StringRelatedField()
 
+    algorithm_interface = AlgorithmInterfaceSerializer(read_only=True)
```

Comment:
Unsure if the job serializers need the interface. It's of no use to the user, just added it for completeness really.